astropy/time/tests/test_basic.py

```diff
@@ -1051,3 +1051,25 @@ def test_to_datetime_pytz():
     for dt, tz_dt in zip(time.datetime, tz_aware_datetime):
         assert tz.tzname(dt) == tz_dt.tzname()
     assert np.all(time == forced_to_astropy_time)
+
+def test_cache():
+    t = Time('2010-09-03 00:00:00')
+    t2 = Time('2010-09-03 00:00:00')
+
+    # Time starts out with no _cache attribute
+    assert not hasattr(t, '_cache')
+
+    # Access the iso format and confirm that the cached version is as expected
+    t.iso
+    assert t.cache['format']['iso'] == t2.iso
+
+    # Access the TAI scale and confirm that the cached version is as expected
+    t.tai
+    assert t.cache['scale']['tai'] == t2.tai
+
+    # New Time object after scale transform does not have a cache yet
+    assert not hasattr(t.tt, '_cache')
+
+    # Clear the cache
+    del t.cache
+    assert not hasattr(t, '_cache')
```

Comment:
Maybe also `assert not t.cache` (I know this creates the cache, but checks it is empty).